aviary/mission/gasp_based/ode/test/test_breguet_cruise_ode.py

```diff
@@ -32,6 +33,8 @@ def setUp(self):
             Dynamic.Atmosphere.MACH, np.array([0, 0]), units="unitless"
         )
 
+        setup_model_options(self.prob, aviary_options)
```

Comment:
I do wonder why these unit tests worked before without passing the model options, but they failed after your changes.